src/js_native_api_v8.cc

```diff
@@ -242,6 +242,8 @@ class RefBase : protected Finalizer, RefTracker {
   static inline void Delete(RefBase* reference) {
     reference->Unlink();
     if ((reference->RefCount() != 0) ||
+        (reference->RefCount() == 0 &&
```

Comment:
`RefCount()` is unsigned, so `reference->RefCount() == 0` is always true here, right?

Reply:
That's true. This can be simplified to the second half of the `&&`.